output.go

```diff
@@ -46,7 +46,28 @@ func Output(w io.Writer, g *Generator, pkg string) {
 
 	for _, k := range getOrderedStructNames(structs) {
 		s := structs[k]
-		if s.GenerateCode {
+		generateCode := false
+		if !skipValidationFlag {
+			for _, f := range s.Fields {
+				if f.Required {
```

Comment:
This is saying that `generateCode = true` if at least one field is required. This is because the code at L139 is the one responsible for the actual check for **each** field. We just need to make sure that the code is called.

Reply:
This is setting `generateCode` to true if at least one field is required, *and* we are not skipping validation (line 50).